src/component/RocketProfile.js

```diff
@@ -0,0 +1,25 @@
+/* eslint-disable */
+import React from "react";
```

Comment:
- [ ] For best practices and as required for all Microverse projects, kindly make sure you fix your linters errors and not disable them.